core/idl/wallet/tezos/tezos_like_wallet.djinni

```diff
@@ -158,7 +158,7 @@ TezosConfiguration = interface +c {
 TezosConfigurationDefaults = interface +c {
 	const TEZOS_DEFAULT_API_ENDPOINT: string = "https://explorers.api.live.ledger.com";
 	const TEZOS_DEFAULT_API_VERSION: string = "v3";
-	const TEZOS_OBSERVER_NODE_ENDPOINT_S2: string = "https://s2.tezos.com";
+	const TZSTATS_API_ENDPOINT: string = "https://api.tzstats.com/explorer";
```

Comment:
does https://explorers.api.live.ledger.com points to this? because in current live logic, we will **always** set the endpoint to https://explorers.api.live.ledger.com

Reply:
What ?! 
No it won't, when you set `BLOCKCHAIN_EXPLORER_ENGINE` to `TZSTATS_API` we will set the endpoint as we should, so you don't need to set anything else from your side !